overlord/snapstate/handlers.go

```diff
@@ -44,6 +44,7 @@ import (
 	"github.com/snapcore/snapd/snap"
 	"github.com/snapcore/snapd/store"
 	"github.com/snapcore/snapd/strutil"
+	"github.com/snapcore/snapd/timings"
```

Comment:
I would prefer the changes to this one to be done in their own PR

Reply:
Will do, removed from this PR.